src/client/monitor/Monitor.tsx

```diff
@@ -65,12 +70,14 @@ export function Monitor({fullScreen, requestFullScreen}: MonitorProps): ReactEle
   const traysAdded = !isEmpty(trays)
   const success = isEmpty(projects)
 
-  const monitorClassNames = cn(styles.monitor, {
-    [styles.fullscreen]: fullScreen
-  })
+  document.onmousemove = () => {
```

Comment:
Using `document.onmousemove` is bad, it breaks React encapsulation meaning the handler isn't going to be cleaned up when the component unmounts. Which means a ton of warnings would be generated on every other page as you'd be trying to update state on an unmounted component!